pyproject.toml

```diff
@@ -36,14 +36,14 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "^3.9"
 
 numpy = "~1.26.0"  # also in geoh5py
 
 ## dependencies from Git repositories
 #------------------------------------
-#geoh5py = {version = "~0.10.0-alpha.1", allow-prereleases = true}
-geoh5py = {git = "https://github.com/MiraGeoscience/geoh5py.git", rev = "develop"}
+geoh5py = {version = ">=0.9.1, <0.11", allow-prereleases = true}
```

Comment:
higher level package can decide to use param-sweeps with geoh5 0.9 or 0.10

Causes less trouble with conda-lock than hard-locking on develop branch